tests/snapshots/build-p.js.md

```diff
@@ -771,7 +771,7 @@ Generated by [AVA](https://ava.li).
           logger: {
             level: 0,
             name: 'untest',
-            workspace: '.',
+            pathRegExp: /\/Users\/daniel\.dembach\/src\/untool\/untool/g,
```

Comment:
This should not contain absolute paths because it will break on other machines..